### PR TITLE
Moving containerattached resource to v1beta1 version

### LINKED ISSUE #207: Moving containerattached(1) resource to v1beta1 version
Moving `containerattached` 1 resources to v1beta1 version
- [ ] [google_container_attached_cluster](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_attached_cluster)

### PR DESCRIPTION
**Pull request by Catramen**
Add containerattachedresource to v1beta1 version

Fixes #207

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

How has this code been tested
create eks cluster with eksctl, apply manifests to the eks cluster and created containerattached cluster resource. Verified the cluster is attached successfully in GCP console.

### CODE REVIEW

File: examples/containerattached/cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    distribution: aks
+    fleet:
+    - project: projects/my-project-number
+    location: us-west1
+    oidcConfig:
+    - issuerUrl: https://oidc.issuer.url
+    platformVersion: 1.23.0-gke.2
+    project: my-project-id
+
+---
```
Comment (sergenyalcin) [inline examples/containerattached/cluster.yaml:21]: Could you please remove these redundant dashes? It causes the failure of publish-artifacts job.

**Review by sergenyalcin — COMMENTED** [review]
Thanks @Catramen I left a comment. 

Apart from this comment, I will trigger an end to end test for this resource via a comment. This test will use the example in this PR. What do you think about this is ready for testing or not? I mean the values in the manifest are valid or not? For example project field and fleet.project field. Could you please check it? After the example is ready for test, we can trigger the e2e job.

**Comment by turkenf** [comment]
Require status checks changed could you please rebase this PR?

**Comment by jeanduplessis** [comment]
Due to a lack of activity on this PR, we are proceeding with closing it for now. If you are interested in seeing this PR through or believe it was closed in error please re-open the PR and add a comment to it.

**Pull request closed without merging**